README.md

```diff
@@ -54,3 +54,89 @@ This app is essentially a frontend microservice and should not be deployed by it
 To test ET3 in full, including RTF upload and form submission, use the [ET Full System](https://github.com/ministryofjustice/et-full-system).
 Before running the full system ensure ET3 is set to the branch or commit you wish to test.
 Please see the README for further details.
+
+## Environment Variables
+
+### DB_PORT
+
+Used to specify the database port.
+ 
+For example, used within the docker-compose configuration.
+
+### S3_PORT
+
+Used to specify the minio port.
+ 
+For example, used within the docker-compose configuration.
+
+### AZURE_STORAGE_BLOB_PORT
```

Comment:
Again, nothing to do with azure needs to be configurable - you are only using it within your test suite, not the app itself.  Maybe an exception to that rule is the port - just in case the person running the test cannot use port 10000 ?